app/src/main/java/com/liato/bankdroid/WebViewActivity.java

```diff
@@ -78,6 +78,7 @@ public void onProgressChanged(WebView view, int progress) {
                     Runnable runnable = new Runnable() {
                         public void run() {
 //                            activity.hideProgressBar();
+                            if (mFirstPageLoaded) mWebView.setBackgroundColor(-1); // -1 == Color.WHITE
```

Comment:
Instead of setting the color int directly, could you maybe add the color to `res/values/colors.xml` and then refer to it as `@colors/white` for example